tcms/testruns/templates/testruns/get.html

```diff
@@ -109,13 +109,77 @@ <h2 class="card-pf-title" style="text-align: left">
         </div>
     </div> <!-- /row -->
 
+    <div class="row row-cards-pf">
+        <div class="col-xs-12 col-sm-12 col-md-12">
+            <div class="card-pf card-pf-accented">
+                <h2 class="card-pf-title">
+                    {% trans 'Test executions' %}
+                </h2>
+                <div class="card-pf-body">
+                    <div id="test-executions-container" class="list-group tree-list-view-pf"></div>
+                </div>
+            </div>
+        </div>
+    </div> <!-- /row -->
+
     <div class="row row-cards-pf">
         <div class="col-xs-12 col-sm-6 col-md-6">
             {% include 'include/attachments.html' %}
         </div>
     </div> <!-- /row -->
 </div>
 
+<template id="test-execution-row">
```

Comment:
Move the template tag inside the card div. Will put the template right where it will be rendered and make it easier to find things when browsing the code.

Also makes it easier to spot if the HTML snippet follows the same structure as in other pages. In the TP page refactoring that proved to be a challenge.